lib/hacluster.pm

```diff
@@ -974,49 +976,55 @@ Calculates start time delay after node is fenced.
 Prevents cluster failure if fenced node restarts too quickly.
 Delay time is used either if specified in sbd config variable "SBD_DELAY_START"
 or calculated:
-"corosync_token + corosync_consensus + SBD_WATCHDOG_TIMEOUT * 2"
+"corosync token timeout + consensus timeout + pcmk_delay_max + msgwait"
 Variables 'corosync_token' and 'corosync_consensus' are converted to seconds.
+For diskless SBD pcmk_delay_max is set to static 30s.
 =cut
 
 sub calculate_sbd_start_delay {
-    my %params;
+    my ($sbd_parameters) = @_;
+    my %params = $sbd_parameters ? %$sbd_parameters : (
```

Comment:
What is the idea of %params? What is the idea of all the `corosync-cmapctl` calls?
What I understand is that `calculate_sbd_start_delay` can work with `sbd_parameters` fully provided by the caller or by using values currently in use in the SUT. 
Is it right?

Reply:
Yes. The 'calculate_sbd_start_delay' was previously used by on prem tests only so it would get the required HA parameters directly from SUT using 'script_output'. The %params was the collection of gathered parameters. 
Since public cloud tests cannot use 'script_output' I created the option to provide those values by caller.